app/lib/TestFeedback.scala

```diff
@@ -49,6 +63,8 @@ object TestFeedback {
  * @param commit SHA
  * @param testResult build result
  * @param buildId build ID
+ * @param screencastId Remote Web Driver session ID
  */
 case class TravisTestResult(repoSlug: String, commit: String,
-                            testResult: String, buildId: String)
+                            testResult: String, buildId: String,
+                            screencastId: String)
```

Comment:
It might be better to pass through the full url here, rather than an id which Prout has to know how to translate...